cmd/telemeter-server/main.go

```diff
@@ -242,30 +241,45 @@ func (o *Options) Run() error {
 			Timeout:   20 * time.Second,
 			Transport: telemeter_http.NewInstrumentedRoundTripper("authorize", transport),
 		}
+	}
 
-		if o.OIDCIssuer != "" {
-			provider, err := oidc.NewProvider(ctx, o.OIDCIssuer)
-			if err != nil {
-				return fmt.Errorf("OIDC provider initialization failed: %v", err)
-			}
+	forwardClient := http.Client{
+		Timeout: 5 * time.Second,
+		Transport: telemeter_http.NewInstrumentedRoundTripper("forward",
+			&http.Transport{
```

Comment:
we should reuse the default transport we created earlier, simply named `transport`, for this since it has the same options set but most importantly because it respects the `verbose` setting